programs/CMakeLists.txt

```
@@ -97,6 +103,12 @@ else()
     message(STATUS "Local mode: OFF")
 endif()
 
+if (ENABLE_CLICKHOUSE_SELF_EXTRACTING)
+    message(STATUS "Self-extracting executable: ON")
+else()
+    message(STATUS "Self-extracting executable: OFF")
+endif()
```

I see the same thing is done for other variables but why not just
```suggestion
message(STATUS "Self-extracting executable: ${ENABLE_CLICKHOUSE_SELF_EXTRACTING}")
```